app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -402,6 +404,35 @@ public void setCurrentSession(int aId) {
             mCurrentSession.setActive(true);
     }
 
+    public String setRegion(String aRegion) {
+        Log.d(LOGTAG, "SessionStore setRegion: " + aRegion);
+        String result = "worldwide";
+        if (mCurrentSession == null || aRegion == null) {
+            return result;
+        }
+        State state = mSessions.get(mCurrentSession.hashCode());
```

Comment:
is this necessary, or is there a better way?

Reply:
Isn't region a app wide setting not bound to a specific session? I guess we could handle it a SessionStore field?